src/node_builtins.cc

```diff
@@ -136,6 +136,7 @@ BuiltinLoader::BuiltinCategories BuiltinLoader::GetBuiltinCategories() const {
         "internal/quic/quic", "internal/quic/symbols", "internal/quic/stats",
         "internal/quic/state",
 #endif             // !NODE_OPENSSL_HAS_QUIC
+        "quic",    // Experimental.
```

Comment:
Shouldn't we expose it only when Node.js is compiled with QUIC support, and the runtime flag `--experimental-quic` is passed?